Content.Shared/Movement/Pulling/Systems/PullingSystem.cs

```diff
@@ -255,7 +255,9 @@ private void StopPulling(EntityUid pullableUid, PullableComponent pullableComp)
 
             if (TryComp<PhysicsComponent>(pullableUid, out var pullablePhysics))
             {
-                _physics.SetFixedRotation(pullableUid, pullableComp.PrevFixedRotation, body: pullablePhysics);
+                // check if we even need to set the FixedRotation to save a dirty call
+                if (pullablePhysics.FixedRotation != pullableComp.PrevFixedRotation)
+                    _physics.SetFixedRotation(pullableUid, pullableComp.PrevFixedRotation, body: pullablePhysics);
```

Comment:
This should be handled in engine not here.

Reply:
Which part specifically should be handled in the engine? Because this code (the part setting Fixed Rotation) existed here even before I touched it

Reply:
you know what, nevermind, I get what you mean
I just checked the SetFixedRotation code and it already makes that check so I didn't need to do this in the first place